lib/chrome.js

```diff
@@ -51,7 +51,7 @@ class Chrome extends EventEmitter {
         options = options || {};
         this.host = options.host || defaults.HOST;
         this.port = options.port || defaults.PORT;
-        this.secure = !!(options.secure);
+        this.secure = !!(options.secure) || !!(options.connectOptions?.secure);
```

Comment:
Why do we need to pass secure flag in two places?
Its better  to use existing options.secure flag.

Reply:
> Why do we need to pass secure flag in two places? Its better to use existing options.secure flag.

For CDP.Version, CDP.Protocol etc. it's possible to make a call like this:
const tls_opt = {
    secure: true,
    key: ...,
    cert: ...,
    ca: ...,
    checkServerIdentity: () => undefined,
}
let ver = await CDP.Version(tls_opt);

For CDP to work with TLS we need to pass separate ` secure:true` flag: 
const cdp = await CDP({
        secure:true,
        connectOptions: {
           agent: new https.Agent(tls_opt),
           ...tls_opt
        }
    });
    
  to stay backward compatible and not to have to pass ` secure:true` flag separately I check if `options.connectOptions?.secure` is there as well

Reply:
Alternative here is to move `tls_opt` from `connectOptions` so to call it like this:
```
const cdp = await CDP({
        ...tls_opt,
        connectOptions: {
           agent: new https.Agent(tls_opt),
        }
    });
```

Reply:
You don't need to set `connectOptions.secure`.
`secure` flag is passed to options from `this.secure` value.
Revert this line.

Reply:
> You don't need to set `connectOptions.secure`. `secure` flag is passed to options from `this.secure` value. Revert this line.

Yes, I just prefer to take this.secure either from constructor options or from connectOptions.secure 
so it is in line with CDP.Version call. 
```
let ver = await CDP.Version(tls_opt);
const cdp = await CDP({ 
   connectOptions: { 
      agent: new https.Agent(tls_opt), 
      ...tls_opt 
   } 
});
```
looks better than 
```
let ver = await CDP.Version(tls_opt);
const cdp = await CDP({ 
   **secure:true,** 
   connectOptions: { 
      agent: new https.Agent(tls_opt), 
      ...tls_opt 
   } 
});
```

Reply:
> You don't need to set connectOptions.secure
fixed